contrib/cirrus/enable_ssh.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -xeo pipefail
+
+dnf install openssh-server openssh-clients -y
```

Comment:
This shouldn't be necessary on the VMs, pretty sure they already have ssh installed (maybe not running though).

Reply:
Leftover cruft from me flaying at containers vs. VMs.